Jetcaster/core/src/main/java/com/example/jetcaster/core/player/EpisodePlayer.kt

```diff
@@ -102,4 +102,8 @@ interface EpisodePlayer {
      * Rewinds a currently played episode by a given time interval specified in [duration].
      */
     fun rewindBy(duration: Duration)
+
+    fun increaseSpeed()
```

Comment:
Maybe this should accept a `Duration`? It can default to half a second. Same comment for `decreaseSpeed`

Reply:
Done